sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/TransactionManager.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus.implementation;
+
+import com.azure.messaging.servicebus.ServiceBusTransactionContext;
+import reactor.core.publisher.Mono;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Provides API to manage transaction on Service Bus namespace.
+ */
+public interface TransactionManager {
```

Comment:
Is there a reason for having this interface instead of an implementation? Will it have more than 1 implementation?